Writing/Question tree.md

```diff
@@ -0,0 +1,25 @@
+---
+tags: writing, consulting, research
+author: Nguyen Xuan Anh
+---
+
+# What is a question tree?
+
+The purpose of a question tree is to brainstorm questions and identify research approaches to gather content about the client’s problem space. The idea is to generate a tree outline of important and miscellaneous questions regarding the problem to explore what business domains and which stakeholders to ask in order to answer our questions.
+
+In consulting, this eventually leads into a decision tree to decide critical business decisions, in which may be presented in a deliverable. For the purposes of this study group, we won’t need to create decision trees, as we want to make sure we focus on software solutions rather than business management.
```

Comment:
I think mentioning the study group here could cause confusion for some readers. Maybe we should change the second sentence to be a bit general?